hivemind_bus_client/client.py

```
@@ -102,8 +102,13 @@ def __init__(self, key: Optional[str] = None,
                  binarize: bool = True,
                  identity: NodeIdentity = None,
                  internal_bus: Optional[OVOSBusClient] = None,
-                 bin_callbacks: BinaryDataCallbacks = BinaryDataCallbacks()):
+                 bin_callbacks: BinaryDataCallbacks = BinaryDataCallbacks(),
```

Ruff (0.8.2)

Do not perform function call `BinaryDataCallbacks` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)